config/initializers/omniauth.rb

```diff
@@ -6,6 +6,7 @@
     prompt: "select_account",
     image_aspect_ratio: "square",
     # we're displaying at 80 pixels, this is for high density ("Retina") displays
-    image_size: 160
+    image_size: 160,
+    hd: ENV["OAUTH2_RESTRICTED_DOMAIN"] || ""
```

Comment:
Actually my bad, we have `ORIENTATION_EMAIL_WHITELIST` for this purpose: https://github.com/orientation/orientation/blob/master/.env.example#L8